lib/puppetfile-resolver/models/module_specification.rb

```diff
@@ -7,11 +7,7 @@
 module PuppetfileResolver
   module Models
     class ModuleSpecification
-      attr_accessor :name
-      attr_accessor :owner
-      attr_accessor :version
-      attr_accessor :origin # Same as R10K module :type
-      attr_accessor :resolver_flags
+      attr_accessor :name, :owner, :version, :origin, :resolver_flags # Same as R10K module :type
```

Comment:
```suggestion
      attr_accessor :name, :owner, :version, :origin, :resolver_flags # origin attr same as R10K module :type
```
judging by the diff this comment was for the `:origin` attribute, maybe update so we dont cause some confusion?